modules/claims_api/spec/requests/v0/claims_request_spec.rb

```diff
@@ -52,6 +52,7 @@
       it 'shows a single errored Claim with an error message', run_at: 'Wed, 13 Dec 2017 03:28:23 GMT' do
         create(:auto_established_claim,
                auth_headers: { some: 'data' },
+               source: 'TestConsumer',
```

Comment:
Should this be consistent throughout the specs, some places have "TestConsumer" and others have "Abe Lincoln" -- is this necessitated by the spec or arbitrary? Want to avoid the mystery guest smell is all

Reply:
its arbitrary. it only matters here because this change requires source of the requester to be the same as the source that created the claim initially.

source was already these different values for different specs, but i can try to sync them up if thats worthwhile. i could see some small benefit to leaving it somewhat disparate so that the "test" of the requester source matching the creator source is a bit less likely to be broken but magically still work. but i can definitely see where you're coming from with the code smell.

happy to do whatever here